src/SFML/Window/Unix/GlxContext.cpp

```diff
@@ -172,6 +172,9 @@ m_ownsWindow(false)
 ////////////////////////////////////////////////////////////
 GlxContext::~GlxContext()
 {
+    // Notify unshared OpenGL resources of context destruction
+    // cleanupUnsharedResources();
+
```

Comment:
This code is inactive, is this intended?

Reply:
Yes it is intended, I just checked the method does not exist in 2.4, or did have another name.

Reply:
Then better to not add it. If it wasn't there and isn't used now, it shouldn't be added as commented out code 😉

Reply:
It's done !